src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IPatternSwitchCase.cs

```diff
@@ -471,7 +471,7 @@ void M(object x)
             string expectedOperationTree = @"
 IPatternCaseClauseOperation (Label Id: 0) (CaseKind.Pattern) (OperationKind.CaseClause, Type: null, IsInvalid) (Syntax: 'case typeof(X):')
   Pattern: 
-    IConstantPatternOperation (OperationKind.ConstantPattern, Type: null, IsInvalid, IsImplicit) (Syntax: 'case typeof(X):') (InputType: System.Type)
+    IConstantPatternOperation (OperationKind.ConstantPattern, Type: null, IsInvalid, IsImplicit) (Syntax: 'typeof(X)') (InputType: System.Type)
```

Comment:
Is this a breaking change to IOperation?

Reply:
I think of it as a bug fix.  A constant pattern should have the syntax of an expression, but this one used to have the syntax of a case label.

---
In reply to: [360582302](https://github.com/dotnet/roslyn/pull/40503#discussion_r360582302) [](ancestors = 360582302)